src/lib/common/include/sol-platform.h

```diff
@@ -74,7 +74,7 @@ extern "C" {
  * When this time is reached, stops the * chunk reading and allow mainloop to run again.
  * This keeps interactivity.
  */
-#define CHUNK_MAX_TIME_NS (20 * (NSEC_PER_MSEC))
+#define CHUNK_MAX_TIME_NS (20 * (1000000000ULL))
```

Comment:
Some platforms (such as Riot) are still defining `time_t` to be 32-bit, so the `ULL` can't be used without a check.